install_config/cluster_metrics.adoc

```diff
@@ -112,11 +112,11 @@ API
 ----
 
 . Before it can deploy components, the *metrics-deployer* service account must
-also be granted the `edit` permission for the *openshift-infra* project:
+also be granted the `edit` permission for the *openshift-metrics* project:
```

Comment:
I think that's a quirk of how the metrics deployment process works -- it's not an inherent need of Heapster.